### PR TITLE
feat: removes personalized FOLLOWED_ARTISTS aggregation, prep for @cacheable

### PR DESCRIPTION
**Pull request by mzikherman**
In order to be able to properly cache grid/filter queries - we needed to figure something out with the personalized 'followed artists' information.

One thing was easy - when your filter query _includes_ that option, naturally we don't want to cache. That was added in 97cb651f2a861b786c216ec14c9ea1996252b6ae.

The other thing was a bit tricky and required a judgement call. We fetch a `FOLLOWED_ARTISTS` aggregation up front as part of the initial filter call. This means that request _must_ skip all cache - both our CDN _and_ any data loader caching in MP. That request has to hit ElasticSearch.

And for what purpose? It purely returns a count....which decides whether a checkbox is enabled or disabled.

I think it's not worth it. Let's just enable/disable the checkbox based on logged in. I went in and removed that property/behavior from the artwork filter.

### CODE REVIEW

File: src/Components/ArtworkFilter/ArtworkFilters/ArtistsFilter.tsx [hunk 1]
```diff
@@ -109,14 +109,13 @@ export const ArtistsFilter: FC<ArtistsFilterProps> = ({ expanded, fairID }) => {
...
   const isFollowedArtistCheckboxSelected =
     !!user && includeArtworksByFollowedArtists
-  const followedArtistArtworkCount = filterContext?.counts?.followedArtists ?? 0
   const hasSelection = artistIDs.length > 0 || isFollowedArtistCheckboxSelected
 
   return (
     <FilterExpandable label={label} expanded={hasSelection || expanded}>
       <Flex flexDirection="column" gap={2} mt={1}>
         <Checkbox
-          disabled={!followedArtistArtworkCount}
```
Comment (mzikherman) [inline src/Components/ArtworkFilter/ArtworkFilters/ArtistsFilter.tsx:119]: This was it. The only place the count was used!

File: src/System/Relay/isRequestCacheable.ts [hunk 2]
```diff
@@ -15,3 +17,10 @@ export const hasNoCacheParamPresent = url => {
 
   return false
 }
+
+// Important - Add any new personalized argument checks to this list. That way, logged-in queries
```
Comment (mzikherman) [inline src/System/Relay/isRequestCacheable.ts:21]: This was just my initial stab - we have this one case we want to cover - when this argument is present and truth-y.

I figure we can refine this lever if/when we have more.

File: src/Apps/Tag/tagRoutes.tsx [hunk 3]
```diff
@@ -31,10 +31,6 @@ export const tagRoutes: RouteProps[] = [
         "ARTIST",
       ]
 
-      if (!!props.context.user) {
-        aggregations.push("FOLLOWED_ARTISTS")
-      }
```
Comment (damassi) [inline src/Apps/Tag/tagRoutes.tsx:36]: 🙏 🙏 🙏

File: src/Components/ArtworkFilter/ArtworkFilters/ArtistsFilter.tsx [hunk 4]
```diff
@@ -109,14 +109,13 @@ export const ArtistsFilter: FC<ArtistsFilterProps> = ({ expanded, fairID }) => {
...
+          disabled={!user}
```
Comment (damassi) [inline src/Components/ArtworkFilter/ArtworkFilters/ArtistsFilter.tsx:118]: 🎉

Comment (damassi) [reply src/Components/ArtworkFilter/ArtworkFilters/ArtistsFilter.tsx:118]: You could even do something like:

```tsx
{user && (
  <Checkbox ... 
)
```

Might be better to hide it altogether? But either way is 👍

**Review by damassi — APPROVED** [review]
Much simpler 👍

File: src/Components/ArtworkFilter/ArtworkFilters/ArtistsFilter.tsx (hunk 4 above)
Comment (mzikherman) [reply src/Components/ArtworkFilter/ArtworkFilters/ArtistsFilter.tsx:118]: Eh, I'll keep it as-is. Very unobtrusive, and the disabled state is nice.

**Pull request merged**